bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -856,16 +968,16 @@ int handleNewWindowRequested(long pView, long pArgs) {
 			WindowEvent openEvent = new WindowEvent(browser);
 			openEvent.display = browser.getDisplay();
 			openEvent.widget = browser;
-			openEvent.required = false;
+			openEvent.required = true;
 			for (OpenWindowListener openListener : openWindowListeners) {
 				openListener.open(openEvent);
 				if (browser.isDisposed()) return;
 			}
 			if (openEvent.browser != null && !openEvent.browser.isDisposed()) {
 				WebBrowser other = openEvent.browser.webBrowser;
 				args.put_Handled(true);
-				if (other instanceof Edge) {
-					args.put_NewWindow(((Edge)other).webView.getAddress());
+				if (other instanceof Edge edge) {
```

Comment:
Please use proper name like `otherEdge`, as now it is unclear that the next line accesses some _other_ edge instance.

Reply:
Alright